_posts/2021-08-02-debezium-1-7-alpha1-released.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  Debezium 1.7.0.Alpha1 Released
+date:   2021-08-02
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: jpechane
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Alpha1*!
+
+With the summer in a full-swing, this release brings additional improvements to the Debezium Oracle connector but also to the others as well.
+
++++<!-- more -->+++
+
+== Alternative Oracle Connector LogMiner processors
+
+The Oracle connector can be memory-heavy in case of really large traffic with parallel large transactions in progress.
+All these changes in progress need to be kept in memory which might be impossible so in previous versions it was necessary to remove the really old transaction out of the picture.
```

Comment:
Instead of "impossible", I'd suggest to describe it along the lines of large memory consumption.